docker/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+  zookeeper:
+    image: zookeeper:3.4.9
```

Comment:
Maybe it is better to use latest zk 3.5.3-beta

Reply:
I remove the tag to the zookeeper version to fetch always the latest